src/server/routes/projects.js

```diff
@@ -140,39 +140,41 @@ var sendProjectTo = function(project, res) {
 const TRANSPARENT = [0,0,0,0];
 
 var padImage = function (buffer, ratio) {  // Pad the image to match the given aspect ratio
-    var lwip = require('lwip');
-    return Q.ninvoke(lwip, 'open', buffer, 'png')
+    let deferred = Q.defer();
+    let handleImageBuffer = (err, imgBuf) => {
+        if (err) {
+            deferred.reject(err);
+        } else {
+            deferred.resolve(imgBuf);
+        }
+    };
+    Jimp.read(buffer)
         .then(image => {
-            var width = image.width(),
-                height = image.height(),
+            var width = image.bitmap.width,
+                height = image.bitmap.height,
                 pad = Utils.computeAspectRatioPadding(width, height, ratio);
-
-            return Q.ninvoke(
-                image,
-                'pad',
-                pad.left,
-                pad.top,
-                pad.right,
-                pad.bottom,
-                TRANSPARENT
-            );
-        })
-        .then(image => Q.ninvoke(image, 'toBuffer', 'png'));
+            // round paddings to behave like lwip
+            let wDiff = parseInt((2*pad.left));
+            let hDiff = parseInt((2*pad.top));
+            image
+                .contain(width + wDiff, height + hDiff)
+                .getBuffer(Jimp.AUTO, handleImageBuffer);
+        });
+    return deferred.promise;
 };
 
-
 var applyAspectRatio = function (thumbnail, aspectRatio) {
     var image = thumbnail
         .replace(/^data:image\/png;base64,|^data:image\/jpeg;base64,|^data:image\/jpg;base64,|^data:image\/bmp;base64,/, '');
     var buffer = new Buffer(image, 'base64');
 
-    if (aspectRatio && typeof lwip !== 'undefined') {
+    if (aspectRatio && typeof Jimp !== 'undefined') {
```

Comment:
We may not need to check if Jimp is defined (since it isn't compiling any external code and should be successful). Not a big deal though